subworkflows/nf-core/fastq_align_dna/main.nf

```diff
@@ -15,8 +15,8 @@ include { SNAPALIGNER_ALIGN as SNAP_ALIGN   } from '../../../modules/nf-core/sna
 
 workflow FASTQ_ALIGN_DNA {
     take:
-        ch_reads            // channel: [mandatory] meta, reads
-        ch_aligner_index    // channel: [mandatory] aligner index
+        ch_reads            // channel: [mandatory] [ val(meta), path(reads)]
```

Comment:
If it's for the sake of normalizing, do you also want to standardize the mandatory comment.
Here it is added before the description but in fasta_dna_index you put it after and inside parentheses.